node-basic/start_actor.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Try to get start script from package.json.
+PACKAGE_START=$(node -e "
+    try {
+        const packageJson = require('./package.json');
+        const startScript = packageJson && packageJson.scripts && packageJson.scripts.start
+            ? packageJson.scripts.start
+            : null;
+
+        console.log(startScript || '');
+    } catch (err) {}
+")
+
+# If not found print warning and fallback to old "node main.js".
+if [ -z $PACKAGE_START ]; then
+    printf '\nWARNING: The npm start script not found in package.json. Using "node main.js" instead. Please update your package.json file. For more information see https://github.com/apifytech/apify-cli/blob/develop/MIGRATIONS.md\n\n'
+fi
+START_SCRIPT=${PACKAGE_START:='node main.js'}
+
+eval $START_SCRIPT
```

Comment:
How about using "exec" command instead? It doesn't start new process, it just replaces the shell. See https://askubuntu.com/questions/525767/what-does-an-exec-command-do